### ISSUE TITLE
problem escaping xml characters in expected output

### ISSUE THREAD

**Issue by DannyMunnik** [unclebob/fitnesse#1145]
Using Fitnesse v 20151230 with DBSlim plugin I run into a problem escaping certain xml codes.
As a result from a query I expect the string '*** MPLS uplink ad00bas01 &lt;-&gt; nl-ams02a-rb5 Gi0/7/0/32 ***'. On the page testpage this shows as expected, but while running the test the &lt; and &gt; are translated to < and > respectively, which causes the test to fail. Normally, I would expect to resolve this issue by escaping the string with !-<string>-!. This results however in the expectation being shown with the < and >, and leading to a failed test.
![image](https://user-images.githubusercontent.com/40764419/42166032-19931512-7e0a-11e8-8203-d881fdb506c9.png)
showing line as being returned by database, lines 2, 3 and 4 as the ways I know to handle escaping.
![image](https://user-images.githubusercontent.com/40764419/42166125-572f0f66-7e0a-11e8-95fd-0e1446289ff0.png)
Showing the result after the test has run.
What do I need to do to have the result being interpreted correctly?

**Comment by Unforgettable631** [comment]
I am not familiar with Db Slim, but what happens with the check adding some WiKi markup? Not sure if this still works.

For example:
![image](https://user-images.githubusercontent.com/26714288/42184546-35800eb0-7e46-11e8-9de3-6acd66a5c622.png)

**Comment by DannyMunnik** [comment]
No, that won't help. Enclosing it with two or three ' leads to the text being shown cursive or bold. It doesn't influence the interpretation of the value.

**Comment by tcnh** [comment]
If you debug the fixture code, what values do you see on the slim side? (in your recordset and in the data coming from the wiki?)
I'd expect no escaping to be necessary. I'm unfamiliar with DB Slim, but when I use another slim fixture to write your string to a file, it lands there without escaping (the fixture does no escaping/unescaping in the process):

```
|script|file fixture                                                              |
|append|*** MPLS uplink ad00bas01 <-> nl-ams02a-rb5 Gi0/7/0/32 ***|to|testfile.txt|
``` 
results in:
![image](https://user-images.githubusercontent.com/2232710/42210075-97c0ac8c-7eb0-11e8-810e-6476d73af15f.png)

**Comment by DannyMunnik** [comment]
Unfortunately I don't have experience with debugging Fitnesse..... But, OK, try to look into that. 
From the latest response, in the line to write the &lt; &gt; have been replaced with < and >. When I have these in the underlying database, and have them in the expectation, it all works OK. however, the situation I run into is that in the underlying database the text is recorded with &lt; and &gt;, which are translated to < and > and therefore are not matching my expectation, which holds the actual text.

**Comment by jediwhale** [comment]
Try this
`: &amp;lt; and &amp;gt;`

**Comment by DannyMunnik** [comment]
unfortunately, that doesn't resolve it as well. Now, also tried by using using the &#60; and &#62; codes, but that doesn't resolve as well
